vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -180,12 +183,12 @@ $global:map = @{
     );
     "c:\akse-cache\win-vnet-cni\" = @(
         # Azure CNI v1 (legacy)
-        "https://acs-mirror.azureedge.net/azure-cni/v1.5.35/binaries/azure-vnet-cni-windows-amd64-v1.5.35.zip",
-        "https://acs-mirror.azureedge.net/azure-cni/v1.6.5/binaries/azure-vnet-cni-windows-amd64-v1.6.5.zip",
+        "https://acs-mirror.azureedge.net/azure-cni/v1.5.38/binaries/azure-vnet-cni-windows-amd64-v1.5.38.zip",
```

Comment:
@sivakami-projects I am not able to download below zip files using the links above. Does that look right? cc @rbtr

https://acs-mirror.azureedge.net/azure-cni/v1.5.38/binaries/azure-vnet-cni-windows-amd64-v1.5.38.zip
https://acs-mirror.azureedge.net/azure-cni/v1.6.12/binaries/azure-vnet-cni-windows-amd64-v1.6.12.zip

Reply:
1.6.12 is not published yet. update the latest publishes version.

Reply:
`v1.5.38` seems to be missing too

Reply:
v1.5.38 has been republished to acs mirror. Works now.